csm_web/frontend/src/components/Course.js

```diff
@@ -87,8 +87,10 @@ class SectionCard extends React.Component {
 
   static propTypes = {
     id: PropTypes.number.isRequired,
-    location: PropTypes.string,
-    time: PropTypes.string.isRequired,
+    spacetime: PropTypes.shape({
```

Comment:
i see a SPACETIME_SHAPE defined in another file, is it possible to reuse that here?

Reply:
Good catch, exported from Section.js to avoid this duplication